File: Java/Driver.java
public class Driver extends Account {
  String vehicleRegistration;
  String photo;
  String insurance;
  String certificateStandardSafety;

  public Driver(String vehicleRegistration, String photo, String insurance, String certificateStandardSafety,
      String name, String document, String email, String password) {
    super(name, document, email, password);
    this.vehicleRegistration = vehicleRegistration;
    this.photo = photo;
    this.insurance = insurance;
    this.certificateStandardSafety = certificateStandardSafety;
  }
}

File: JavaScript/Driver.js
class Driver extends Account {
  constructor(email, password, name, document, photo, vehicleRegistration, insurance, certificateStandardSafety) {
    super(email, password, name, document)
    this.photo = photo
    this.vehicleRegistration = vehicleRegistration
    this.insurance = insurance
    this.certificateStandardSafety = certificateStandardSafety
  }
}